tt_metal/hw/inc/dataflow_api.h

```diff
@@ -557,7 +577,7 @@ std::uint64_t get_noc_addr_helper(std::uint32_t noc_xy, std::uint32_t addr) {
 uint64_t get_dram_noc_addr(const uint32_t id, const uint32_t page_size, const uint32_t bank_base_address, const uint32_t offset = 0, uint8_t noc = noc_index) {
     uint32_t bank_offset_index = interleaved_addr_gen::get_bank_offset_index<true>(id);
     uint32_t bank_index = interleaved_addr_gen::get_bank_index<true>(id, bank_offset_index);
-    uint32_t addr = (bank_offset_index * align(page_size, ALLOCATOR_ALIGNMENT)) + bank_base_address + offset + bank_to_dram_offset[bank_index];
+    uint32_t addr = (bank_offset_index * align(page_size, interleaved_addr_gen::get_allocator_alignment<true>())) + bank_base_address + offset + bank_to_dram_offset[bank_index];
```

Comment:
Is there a reason you use true/flase in lines 580/589 where you use DRAM/ in line 915. I find the latter much more readable